dawn/renderer/utils/sagas.js

```diff
@@ -81,19 +82,16 @@ function* openFile() {
  */
 function* writeFile(filepath, code) {
   yield cps(fs.writeFile, filepath, code);
-  yield put({
-    type: 'SAVE_FILE_SUCCEEDED',
-    code,
-    filepath,
-  });
+  yield put(saveFileSucceeded(code, filepath));
 }
 
+const editorState = (state) => ({
```

Comment:
*.test.js is in `.eslintignore`. Should we remove this so tests are also linted?